src/scripts/scripts/zone/hellfire_citadel/blood_furnace/instance_blood_furnace.cpp

```diff
@@ -104,19 +104,12 @@ struct instance_blood_furnace : public ScriptedInstance
             case 181819: BroggokDoor2GUID = go->GetGUID(); break;
             case 181823: Sewer1GUID = go->GetGUID(); break;
 +           case 181766: 
-
 +                {
-
 +                    Sewer2GUID = go->GetGUID(); 
-
 +					
-
 +                    if (Encounter[2] == DONE)
-
 +                        HandleGameObject(Sewer2GUID, 0);
-
 +                }
-
```

Comment:
okay so the function doesnt work like this, then maybe the flaw is at

            case DATA_KELIDANEVENT:
                if (data == DONE)
                {
                    HandleGameObject(Sewer1GUID, 0); 
                    HandleGameObject(Sewer2GUID, 0);

he only handels the Sewer1GUID but doesnt handle Sewer2GUID.